include/deal.II/integrators/advection.h

```diff
@@ -256,7 +340,128 @@ namespace LocalIntegrators
             }
         }
     }
+    /**
+     * Scalar: Residual for upwind flux at the boundary for weak advection operator. This is the
```

Comment:
Say: Scalar case. (And use the same notation for the cell residuals above, instead of the notation I suggested.)